<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="downloadTab">
     <attribute name="title">
      <string>Download</string>
     </attribute>
     <widget class="QPushButton" name="dlChorusBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>DL from Chorus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dlMasterSheetBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>DL from Master Sheet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dlC3Btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>DL from C3</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="dlText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>771</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <zorder>dlChorusBtn</zorder>
     <zorder>dlMasterSheetBtn</zorder>
     <zorder>dlText</zorder>
     <zorder>dlC3Btn</zorder>
    </widget>
    <widget class="QWidget" name="organizeTab">
     <attribute name="title">
      <string>Organize</string>
     </attribute>
     <widget class="QLabel" name="dlChartCount">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>downloaded charts: 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="cleanChartCount">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>clean charts: 0</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="organizeText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>791</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="unpackBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>47</height>
       </rect>
      </property>
      <property name="text">
       <string>Unpack 'n' Clean</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rmBadBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>331</width>
        <height>47</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove &quot;bad songs&quot;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="convertMidBtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>331</width>
        <height>47</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert .mid to .chart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="convertSongsBtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>331</width>
        <height>47</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert RockBand files to CH</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsTab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLineEdit" name="dirText">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>550</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>download/directory/here/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseDirBtn">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>210</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose Directory</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
